pkg/doc/signature/proof/data.go

```diff
@@ -19,6 +22,32 @@ type signatureSuite interface {
 	GetDigest(doc []byte) []byte
 }
 
+// SignatureHolding defines a representation of signature value.
+type SignatureHolding int
```

Comment:
I'm not finding "Holding" to be an intuitive description of this type.

Reply:
It's a sort of "a type of signature value holder". Maybe `SignatureRepresentation` will sound better? Or "SignatureHoldingType"?

Reply:
probably SignatureRepresentation is better. the word holding confuses me for an enumerated type.

Reply:
Thank you! renamed.